alertmanager/config.go

```diff
@@ -0,0 +1,31 @@
+// promconfig
+// Copyright 2020 Percona LLC
+//
+// Based on Prometheus systems and service monitoring server.
+// Copyright 2015 The Prometheus Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package alertmanager
+
+// AMConfig is the top-level configuration for Alertmanager's config files.
+type AMConfig struct {
```

Comment:
We already have "Alertmanager" in the package name. It will be used as `alertmanager.AMConfig`, and that stutters. Let's call it just Config